### PR TITLE
[UI] Fix Monitor in Moderator interface:

### PR DESCRIPTION
**Pull request by ankitjavalkar**
- Fix indentation
- Add proper corresponding closing <div> tags
- Add closing <span> tags

### CODE REVIEW

File: yaksh/templates/yaksh/user_data.html [hunk 1]
```diff
@@ -1,271 +1,300 @@
...
+         Position: {{ data.profile.position }} <br/>
+         Department: {{ data.profile.department }} <br/>
+         Institute: {{ data.profile.institute }} <br/>
+         {% endif %}
+         Email: {{ data.user.email }} <br/>
+         Date joined: {{ data.user.date_joined }} <br/>
+         Last login: {{ data.user.last_login }}
+      </p>
+   </div>
+<!-- </div> -->
```
Comment (prathamesh920) [inline yaksh/templates/yaksh/user_data.html]: This comment be removed

File: yaksh/templates/yaksh/user_data.html [hunk 2]
```diff
@@ -1,271 +1,300 @@
...
+   {% if data.papers %}
+   <p class="text-center"><a href="{{URL_ROOT}}/exam/manage/gradeuser/{{data.papers.0.question_paper.quiz.id}}/{{ data.user.id }}/{{course_id}}/"  class="btn btn-info ">
+      Grade/correct paper</a>
+   </p>
+   {% for paper in data.papers %}
+   {% if forloop.counter == 2 and data.questionpaperid %}
+   <hr>
+   <U>
```
Comment (prathamesh920) [inline yaksh/templates/yaksh/user_data.html]: Only if there is another commit to follow then this ```u tag``` can be lowercase for consistency purpose

**Comment by prathamesh920** [comment]
LGTM
Can be merged once the PR is updated.

File: yaksh/templates/yaksh/user_data.html (hunk 2 above)
Comment (ankitjavalkar) [reply yaksh/templates/yaksh/user_data.html]: Done

File: yaksh/templates/yaksh/user_data.html (hunk 1 above)
Comment (ankitjavalkar) [reply yaksh/templates/yaksh/user_data.html]: Done

**Pull request merged**